data/human/culture conversations.txt

```diff
@@ -1254,3 +1254,375 @@ mission "Little Vienna in New Austria"
+	word
+		"while a gentle wind stirs the dust on the track."
+		"with the sun's beams lending a warm glow to the cool sky."
+		"with sunlight and swirling mist setting an unpredictable stage."
+		"while a shifting haze fails to hide the crowd's raw anticipation."
+	word
+		" "
+	word
+		"A pulsing neon light flashes "
+		"The starting gun goes off "
+		`A booming voice announces, "It's repulsors on and away we go!" `
+		"In an instant, there's a thunderous explosion "
+	word
+		"at the starting line"
+		"in front of the starting grid"
+	word
+		" and the hovercraft launch forward in a burst of speed."
+		", then engines ignite in a frenzy to send the sleek machines darting into the first lap."
+		" as the racers enter into a chaotic scramble for the lead."
+		" - hovercraft fighting for position as one leaps ahead of the rest."
+	word
+		" "
+	word
+		"An hour in, the racers continue to weave through the course's twisting tracks, "
+		"Midway, the competition transforms into a dynamic dance, "
+		"As the race unfolds, a few hovercraft nearly crash before recovering, "
+	word
+		"each maneuver a blend of precision and raw power."
+		"with daring overtakes and strategic maneuvers."
+		"each moment getting closer to the crescendo of the race's final hour."
+		"as competitors push their vehicles to the limit at every turn."
+
+phrase "sunracer hovercraft victory description"
+	word
+		" is today's victor."
+		" seizes the day with a victory."
+		" wins this race."
+
+phrase "sunracer hovercraft victory exit"
+	word
+		"With the race finished, the crowd gradually disperses, and you wander back to your <model> amid the soft buzz of post-race chatter."
+		"As the stands empty, you slip away through clusters of lingering fans back to the <ship>."
+		"Despite the race being over, the crowd still roars with adrenaline, and you still feel the crowd's excitement as you return to your ship."
+		"The venue quiets as spectators scatter, leaving you to drift back to your <model> in the calm that's followed the thrill of the race."
+
+mission "A Sunracer Hovercraft Racing"
+	minor
+	repeat
+	invisible
+	to offer
+		random < 40
+	source "Sunracer"
+	substitutions
+		"<racename>" "${adjectives} ${racetrack}"
+		"<megaparsec name>" "${civilian}"
+		"<syndicate name>" "${syndicate fighter}"
+		"<delta name>" "${free worlds fighter}"
+		"<deep name>" "${deep fighter}"
+	on offer
+		conversation
+			`${sunracer opening}, a ${sunracer hovercraft adhook}: there's a hovercraft race at the <racename>, just a few hours away by hover rail, or much faster aboard a <model>. ${sunracer hovercraft adcloser}`
+			choice
+				`	(Check it out.)`
+				`	(Skip the race.)`
+					decline
+			action
+				"race length" = 3 + "roll: 2"
+				"syndicate boost" = 2 + "roll: 2"
+				"delta boost" = 4 + "roll: 8"
+				"deep boost" = 6 + "roll: 12"
+				# The ( ( 10 - x + "race length" ) / 10 ) flips to 1 when race length is x+, 0 if less than x. Used to only roll for the length of the track.
```

Comment:
```suggestion
				# The ( ( 10 - x + "race length" ) / 10 ) flips to 1 when race length is x or greater, 0 if it's less than x. Only used to roll for the length of the track.
```

Could also use something like `race length ≥ x` here, but considering that ES barely supports ASCII that might not be the best option.